Adds notes for 4.10.18 release (#46628)
openshift/openshift-docs: release_notes/ocp-4-10-release-notes.adoc

==== Upgrading

To upgrade an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.

[id="ocp-4-10-18"]
=== RHBA-2022:4944 - {product-title} 4.10.18 bug fix and security update

Issued: 2022-06-13

> {product-title} release 4.10.17, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:4944[RHBA-2022:4944] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:4943[RHSA-2022:4943] advisory.

Reviewer: ```suggestion
{product-title} release 4.10.18, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:4944[RHBA-2022:4944] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:4943[RHSA-2022:4943] advisory.
```

Author: Updated! Thanks for catching!

Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:

link:https://access.redhat.com/solutions/6962977[{product-title} 4.10.18 container image list]

[id="ocp-4-10-18-bug-fixes"]
==== Bug fixes

> * Previously, Alibaba Cloud is only supported for Disk volumes larger than 20 GiB. Consequently, attempts to dynamically provision a new volume for a persistent volume claim (PVC) smaller than 20GiB fails. With this update, {product-title} will automatically increase the volume size for a PVC and it will provision volumes at least with 20 GiB in size.

Reviewer: ```suggestion
* Previously, Alibaba Cloud was only supported for disk volumes larger than 20 GiB. Consequently, attempts to dynamically provision a new volume for a persistent volume claim (PVC) smaller than 20GiB failed. With this update, {product-title} will automatically increase the volume size for a PVC and it will provision volumes at least with 20 GiB in size.
```

Author: Updated

 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076671[*BZ#2076671*])

> * Previously, the Ingress Operator had unnecessary logic to remove a finalizer on `LoadBalancer-type` services in previous versions of {product-title}. With this update, the finalizer logic is removed and the Ingress Operator no longer includes this logic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2082161[*BZ#2082161*])

Reviewer: ```suggestion
* Previously, the Ingress Operator had unnecessary logic to remove a finalizer on `LoadBalancer-type` services in previous versions of {product-title}. With this update, the Ingress Operator no longer includes this logic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2082161[*BZ#2082161*])
```

Author: Updated
